pywal/util.py

```diff
@@ -36,16 +36,21 @@ def xrgba(self):
     def rgba(self):
         """Convert a hex color to rgba."""
         return "rgba(%s,%s,%s,%s)" % (*hex_to_rgb(self.hex_color),
-                                      int(self.alpha_num) / 100)
+                                      self.alpha_dec)
 
     @property
     def alpha(self):
         """Add URxvt alpha value to color."""
         return "[%s]%s" % (self.alpha_num, self.hex_color)
 
+    @property
+    def alpha_dec(self):
+        """Export the alpha value as a decimal number in [0, 1]."""
+        return int(self.alpha_num) / 100
+    
```

Comment:
Line 50 has extra whitespace, which is making the CI checks fail.

Reply:
oh thx, done in 0b3158490fa105e18c4da274614d63e186f22f61.

Reply:
Nice, lets hope this gets merged for that sweet kitty term opacity.

Reply:
Cool :+1:, but like I said in the description above: I think we need to fix #495 first. Either through #485 or #496.

Reply:
Alrighty, i just commented on both pull requests, suggesting changes that should hopefully fix the CI tests.